db/migrate/20210107061732_drop_casa_cases_emancipation_options.rb

```diff
@@ -0,0 +1,5 @@
+class DropCasaCasesEmancipationOptions < ActiveRecord::Migration[6.1]
+  def change
+    drop_table :casa_cases_emancipation_options
```

Comment:
This is not safe in prod IF any volunteers have edited their emancipation options- we would lose their data. So before we merge this, we have to check for data in prod- and once it's merged, we have to hurry it into prod before any data is created. You can see the count of casa_cases_emancipation_options here: https://data.heroku.com/dataclips/cnluraqwatwiupkkhkpueonwqcfz

I would prefer a safer migration path. @littleforest what do you think?

Reply:
Since it looks like we just want to add on the primary key that was left off, as well as the `created_at`, `updated_at` timestamps, I would just make those changes without destroying the entire table:

Add primary key:
https://stackoverflow.com/questions/5328528/how-do-i-retroactively-add-a-primary-key-to-my-table-in-rails

Add timestamps:
https://stackoverflow.com/a/43548750/1753903